### PR TITLE
2 new species (domestic sheep and pig)

### PR DESCRIPTION
**Pull request by UlrikeS91**

### CODE REVIEW

File: instances/species/ovisAries.jsonld [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/species/ovisAries",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Species",
+  "definition": "The species 'Ovis aries' (domestic sheep) is a mammal of the artiodactyla (even-toed ungulates) order and genus ovis (wild and domestic sheep).",
```
Comment (lzehl) [inline instances/species/ovisAries.jsonld]: for consistency with the other species this should only resolve to the family.
"The species *Ovis aries* (domestic sheep) belongs to the family of *bovidae* (bovids)."

File: instances/species/susScrofaDomesticus.jsonld [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/species/susScrofaDomesticus",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Species",
+  "definition": "The species 'Sus scrofa domesticus' (domestic pig) is a mammal of the artiodactyla (even-toed ungulates) order and genus sus (wild and domestic pigs).",
```
Comment (lzehl) [inline instances/species/susScrofaDomesticus.jsonld]: "The species *Sus scrofa domesticus* (domestic pig) belongs to the family of *suidae* (suids)."

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/species/ovisAries.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/species/ovisAries.jsonld]: fixed

File: instances/species/susScrofaDomesticus.jsonld (hunk 2 above)
Comment (UlrikeS91) [reply instances/species/susScrofaDomesticus.jsonld]: fixed

**Review by lzehl — APPROVED** [review]

**Pull request merged**